### ISSUE TITLE
Poster: Speaker Series 

### ISSUE THREAD

**Issue by sabrinang** [MozillaFoundation/mozfest-design-2017#37]
### Description:
Design promotional poster for the speakers this year at MozFest. Last year, they we're designed as 5 series to feature a cluster of speakers. Posters should include information such as:

- Time of talk
- Speakers' names
- Speakers' photos
- Title of each speaker's talk
- Location (Walker Space on the ground floor)

Information on speakers here: https://mozillafestival.org/speakers

**Format**
- A2

### Blocked by
- series format

### Previous Applications
<img width="1199" alt="image" src="https://user-images.githubusercontent.com/2492510/29842387-8f0f36e8-8cbd-11e7-9acf-6a15be5ae67a.png">

### Reference from Last Year: 
Google Drive Files: https://drive.google.com/open?id=0B3b1KWRwjxtocHk3Q3QtZ3BnNVU
Github Issue: https://github.com/MozillaFoundation/Mozfest2016_production/issues/167
__________________________
#### Deadline
GanttStart: 2017-09-25
GanttDue: 2017-09-29

**Comment by edrushka** [comment]
All the info is [here.](https://medium.com/mozilla-festival/journalists-activists-hackers-meet-this-years-mozfest-speakers-89018c7d2c97) This is accurate as of Sept 25. Recommend you leave this as late as possible (still meeting print deadline) as things _may_ shift.

**Comment by sabrinang** [comment]
I started the design inspired from the [festival guide pages](https://redpen.io/hp21d8e98e7e54b49c) and iterating on type and hierarchy:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/2492510/31638804-6901c69e-b28a-11e7-9b8a-123bac70fbbc.png">

Redpen here: https://redpen.io/p/xz7d85a1b4a215614f

Here are the files are in this gdrive: https://drive.google.com/drive/folders/0B4Q8pzCpDS_eeHk5eFUxcFRRaFU?usp=sharing

**Comment by sabrinang** [comment]
Assigning to @taisdesouzalessa to help me design the rest

**Comment by sabrinang** [comment]
@marcwalsh Can you confirm if this is the format (A2): https://www.pixartprinting.co.uk/digital-litho-printing/printing-posters/

**Comment by marcwalsh** [comment]
I’m happy with A2!

**Comment by marcwalsh** [comment]
Let's also print these at Ravensbourne :) @sabrinang @taisdesouzalessa 

This is just because we're having multiple posters and not a high print run.

**Comment by taisdesouzalessa** [comment]
Thanks @marcwalsh! Do you know which day we will have the printers available to print them?

**Comment by taisdesouzalessa** [comment]
@sabrinang here are the posters in RedPen: 
https://redpen.io/p/xz7d85a1b4a215614f

I double checked the info but it would be nice to have a last pass to make sure there are no mistakes. Who is revising the copy?

**Comment by sabrinang** [comment]
Thanks @taisdesouzalessa R+ on design from me

@edrushka Can you give a content review for these posters? (see redpen: https://redpen.io/p/xz7d85a1b4a215614f)

**Comment by edrushka** [comment]
These look fabulous to me. R+  CC-ing @KevZawacki for final 👀 on content.

**Comment by KevZawacki** [comment]
@edrushka @sabrinang Oh my, these are gorgeous! Not one edit from me.

**Comment by taisdesouzalessa** [comment]
@marcwalsh here is the PDF:
https://drive.google.com/file/d/0B3b1KWRwjxtoenV0NG1QWmhuekk/view?usp=sharing

**Comment by taisdesouzalessa** [comment]
@marcwalsh I was told the file was too big so what I did was separate the posters into individual ones - we need 4 copies of each:
[dd1.pdf](https://github.com/MozillaFoundation/mozfest-design-2017/files/1422624/dd1.pdf)
[dd2.pdf](https://github.com/MozillaFoundation/mozfest-design-2017/files/1422625/dd2.pdf)
[dd3.pdf](https://github.com/MozillaFoundation/mozfest-design-2017/files/1422626/dd3.pdf)
[dd4.pdf](https://github.com/MozillaFoundation/mozfest-design-2017/files/1422627/dd4.pdf)
[dd5.pdf](https://github.com/MozillaFoundation/mozfest-design-2017/files/1422628/dd5.pdf)
[dd6.pdf](https://github.com/MozillaFoundation/mozfest-design-2017/files/1422629/dd6.pdf)

I'll send to the print email as well.

cc: @sabrinang

**Comment by sabrinang** [comment]
## Post MozFest Documentation
The set of speaker series posters were found outside of the walker space and they looked great with the set of wayfinding banners and floor stickers. 
![mozilla-festival-2017-204](https://user-images.githubusercontent.com/2492510/33099899-640c34b8-cec7-11e7-815d-8a69ba14acd1.jpg)
![mozilla-festival-2017-53](https://user-images.githubusercontent.com/2492510/33099952-8ec9170c-cec7-11e7-9f67-23365da26fee.jpg)
![mozilla-festival-2017-51](https://user-images.githubusercontent.com/2492510/33099997-b1f74cee-cec7-11e7-86df-62e22c72a565.jpg)
![20171027-mozfest-signage-34](https://user-images.githubusercontent.com/2492510/33100263-98187c02-cec8-11e7-8d7c-8d9d6e856c2b.jpg)
